### DISCUSSION TITLE
Chat bubbles

### DISCUSSION THREAD

**Discussion by hardBSDk** [status-im/status-legacy#12616] [Ideas]
Implement an option to enable chat bubbles above other apps (Facebook Messenger-like feature, 
even WhatsApp/Telegram doesn't have that).

https://developer.android.com/guide/topics/ui/bubbles

Code suggestion: (because IDK if Bubbles API is supported on older versions of Android)

https://github.com/sentialx/discord-bubbles (Kotlin)

https://github.com/txusballesteros/bubbles-for-android (Java)

![bubbles_demo](https://user-images.githubusercontent.com/55782009/126927984-9f1ef444-b7a6-4b4f-8fc2-968ec00242ca.gif)

![72634751-7e2e6b80-395b-11ea-9033-a834f1dc61a3](https://user-images.githubusercontent.com/55782009/126928034-e1482fd3-f8b1-40a4-9579-8c2a328ad879.gif)

**Comment by du82** [comment]
Nobody outside of people in India actually use the feature though... Just saying...

It's used lots with Facebook properties, but almost no one uses it outside of it, it's quite annoying.

**Comment by hardBSDk** [comment]
> Nobody outside of people in India actually use the feature though... Just saying...
> 
> It's used lots with Facebook properties, but almost no one uses it outside of it, it's quite annoying.

Maybe, I never saw a open source messenger with this feature, because Facebook Messenger is very bad, people may have had a bad impression of the feature.
